### PR TITLE
chore(PostgreSQL-DB): Configure Authors Haven to use PostgreSQL as DB…

### PR DESCRIPTION
**Pull request by Mrshinezee**
#### What does this PR do?
-Configure the project to use sequelize and postgres

#### Description of Task to be completed?
- Integration of sequelize into the project

#### How should this be manually tested?
- Clone the repo
- cd into the 'lotus-ah'
- Run 'npm install'
- Use the .env.sample format to create a .env file
- Run 'npm run start:dev'

#### What are the relevant pivotal tracker stories?
[#160451587](Configure Authors Haven to use PostgreSQL as DB Engine )

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["es2015"]
+}
```
Comment (KaiserPhemi) [inline .babelrc]: #### Nitpick
Insert a new line at the end of every file.

File: index.js [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
+import express from 'express';
```
[the remark this answers is not in the document]
Comment (KaiserPhemi) [reply index.js:1]: #### Critical
Why do we have this flag ? Please check the configuration file and fix this.

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]

File: .babelrc (hunk 1 above)
Comment (Mrshinezee) [reply .babelrc]: Thanks will fix it

File: index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Mrshinezee) [reply index.js:1]: I refactored the es5 to es6, I am still trying to figure out why houndci is flagging import

**Review by obulaworld — APPROVED** [review]
Looking good to me

**Review by Victor-Ugwueze — APPROVED** [review]
LGTM

**Review by KaiserPhemi — APPROVED** [review]

**Pull request merged**